### ISSUE TITLE
Suggestion: add PrivMX WebMail

### ISSUE THREAD

**Issue by macmusz** [nylira/prism-break#1838]
Hi, I'd like to suggest an addition to the prism-break website - https://privmx.com
Keywords: alternative (non-smtp) mail system, web-based, decentralized, self-hosted, end-to-end encrypted by design, freeware, open-source, independent CONIKS-like system for storing and verifying public keys, end-to-end encrypted web forms, ...

**Comment by Zegnat** [comment]
I flipped through [the architecture description](https://simplito.com/wp-content/uploads/2017/10/PrivMXArchitecture.pdf) / white-paper for a bit, but think I am going to have a good sit down to see what it is all doing exactly before I can give a real opinion. It sure does look interesting, though you’ll have to forgive me for being a bit sceptical as soon as people start talking about reinventing TLS for their own purposes.

Are there any plans for releasing an HTML version of the document? That would’ve surely been easier to read than this PDF.

**Comment by macmusz** [comment]
HTML version of the document: https://simplito.com/en/kb/privmx/

We also are sceptical when somebody wants to change something which is "well-established". Our PrivMX TLS is not a reinvention, it is core TLS protocol with some additional features.

We have got some assumptions about PrivMX architecture and the web mail app and one of them has been to use non-duplex (non-async) transport protocol, with http as the main example. That is why we made some modifications related to the request-response nature of http. It can be said that we have been doing something like "TLS over HTTP" here, a transfer of TLS to higher network layer. One can find similar attempts made by others, for example: https://datatracker.ietf.org/doc/html/draft-friel-tls-over-http-00
Another example of our additions is little different session control. We have been using a new kind of tokens, little different than those which are known from https://tools.ietf.org/html/rfc5077. 
  
TLS is OK :) We are not changing it -- we have been using it in the context of our work.
